solidity/core/package.json

```diff
@@ -24,7 +24,7 @@
     "solhint-plugin-prettier": "^0.0.5",
     "solidity-coverage": "^0.7.14",
     "ts-generator": "^0.1.1",
-    "typechain": "8.0.0",
+    "typechain": "8.1.0",
```

Comment:
It's better if we keep versions of important deps like this consistent across the monorepo, so if this changes consider changing solidity/app/package.json as well.
But @yorhodes mentioned he had a problem with 8.1.0 so pinned it to 8

Reply:
https://github.com/dethcrypto/TypeChain/issues/721

Reply:
Gotcha

Reply:
Removed it for now